### PR TITLE
Appendix: document settings files & folders

### PR DESCRIPTION
**Pull request by ronso0**
move the documentation from the wiki to the manual to make it more discoverable.

### CODE REVIEW

**Comment by Holzhaus** [comment]
The the Mixxx directory location are already mentioned in the "Updating Mixxx" section. Can we merge those somehow. I suggest a "See also" box in the "Updating Mixxx" section.

**Comment by ronso0** [comment]
oh, sure.
box added at the bottom of "Upgrading Mixxx"

File: source/chapters/appendix/settings_folder.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.. include:: /shortcuts.rstext
+
+.. _appendix-settings-files:
+
+The Mixxx Settings Folder
```
Comment (Holzhaus) [inline source/chapters/appendix/settings_folder.rst:5]: Directory

File: source/chapters/appendix.rst [hunk 2]
```diff
@@ -10,5 +10,6 @@ Appendix
    appendix/application_shortcuts
    appendix/commandline_dev_tools
    appendix/mixxx_controls
+   appendix/settings_folder
```
Comment (Holzhaus) [inline source/chapters/appendix.rst:13]: Directory

**Comment by ronso0** [comment]
TODO remove the wiki page

File: source/chapters/getting_started.rst [hunk 3]
```diff
@@ -50,6 +50,8 @@ make a backup of your existing settings just in case.
 Copy this folder to a backup drive or somewhere else on your system to make a
```
Comment (Holzhaus) [inline source/chapters/getting_started.rst:50]:
```suggestion
Copy this directory to a backup drive or somewhere else on your system to make a
```

File: source/chapters/appendix/settings_directory.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+**effects.xml**
+    The current configuration of the 4 effect units, incl. the state of all controls.
+
+**mixxx.cfg**
+    Everything you configured in the Preferences, deck settings, skin settings,
+    AutoDJ configuration, effect routing etc.
+
+**mixxx.log[.NN]**
+    session logs
```
Comment (Holzhaus) [inline source/chapters/appendix/settings_directory.rst:60]: Maybe something like this:

```suggestion
    Logfiles of the last few Mixxx sessions, with ``mixxx.log`` being the most recent one.
```

Comment (ronso0) [reply source/chapters/appendix/settings_directory.rst]: Thanks!

**Comment by ronso0** [comment]
okay, done. thanks @Holzhaus for the review.
I will not rename the branch `settings-directory` though : )

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Pull request merged**